go.mod

```diff
@@ -10,6 +10,7 @@ require (
 	github.com/concourse/retryhttp v0.0.0-20170802173037-937335fd9545
 	github.com/dgrijalva/jwt-go v3.2.0+incompatible // indirect
 	github.com/jessevdk/go-flags v1.4.0
+	github.com/klauspost/compress v1.9.7
```

Comment:
I think you missed a `go mod tidy` 😁  it'd get rid of the datadog version

Reply:
yes i did